specification/network/resource-manager/Microsoft.Network/stable/2023-05-01/firewallPolicy.json

```diff
@@ -559,7 +559,7 @@
           "Create FirewallPolicyRuleCollectionGroup With Web Categories": {
             "$ref": "./examples/FirewallPolicyRuleCollectionGroupWithWebCategoriesPut.json"
           },
-          "Create FirewallPolicyRuleCollectionGroup With http header to insert": {
+          "Create FirewallPolicyRuleCollectionGroup With Custom HTTP headers": {
```

Comment:
'Custom' need not be capitalized. Its also recommended style wise, not to use the resource names or programmatic names in documentation, but to use the plain English names conveying meaning instead like,

FirewallPolicyRuleCollectionGroup -> firewall rule collection group